snakemake_executor_plugin_slurm/__init__.py

```diff
@@ -31,13 +33,51 @@
 
 @dataclass
 class ExecutorSettings(ExecutorSettingsBase):
+    logdir: Optional[str] = field(
+        default=os.path.join(os.path.expanduser("~"), ".snakemake", "slurm_logs"),
+        metadata={
+            "help": """
+                   Per default the SLURM log directory (writing output is
+                   required by SLURM) is '~/.snakemake/slurm_logs'.
+                   This flag allows to set an alternative directory.
+                   """,
+            "env_var": False,
+            "required": False,
+        },
+    )
```

Comment:
I think the default should be in the workdir and not in the users home, like it was before.

Reply:
This was what I posted to be discussed: By changing the default, the clean-up will be enacted _every_ time. Alternatively, a user has to clean up hidden files, eventually. In my experience, this is what will not happen.

We are talking about SLURM logs, here. Log files, which do not contain substantial information. As Snakemake submits itself, it will mostly have the same information as printed on the terminal — particularly for failed jobs in the case of workflows using the `log` directive, which we ought to expect. The number of log files can be enormous. They may have a direct impact on a file quota in a work space (if enabled). We do not need to keep any in the long run. And we should think about core facilities or users who run different workflows and potentially accumulate this redundant information at different places. 

Granted: Deleting logs of successful runs makes a huge difference, already. Granted: This is a deviation from the current default.

If you still want to keep the current default, I will alter this PR accordingly.